app/controllers/hostgroups_controller.rb

```diff
@@ -76,11 +75,10 @@ def destroy
   end
 
   def current_parameters
-    Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
```

Comment:
This function actually doesn't work right now for hostgroups, always returning 404 or 500 when changing any selction on the hostgroup which triggers the ajax call to this function. Right now hostgroups don't display any inherited parameters.